internal/cmd/egctl/testdata/config/out/out.all.json

```diff
@@ -1,1513 +1,2316 @@
 {
-	"default": {
```

Comment:
why are these deleted

Reply:
It is not deleted, just change the indent, we write it by hand before, now it is overidden by codes.